### PR TITLE
docs: add release notes for all other languages

### PR DESCRIPTION
**Pull request by aslushnikov**

### CODE REVIEW

File: docs/src/release-notes-csharp.md [hunk 1]
```diff
@@ -5,6 +5,64 @@ title: "Release notes"
...
+- Web-First assertions
+- New options for methods [`method: Page.screenshot`], [`method: Locator.screenshot`] and [`method: ElementHandle.screenshot`]:
+  * Option `animations: "disabled"` rewinds all CSS animations and transitions to a consistent state
+  * Option `mask: Locator[]` masks given elements, overlaying them with pink `#FF00FF` boxes.
+- [Trace Viewer](./trace-viewer) now shows [API testing requests](./src/test-api-testing).
+- [`method: Locator.highlight`] visually reveals element(s) for easier debugging.
+
+### Announcements
+
+- v1.20 is the last release that ships WebKit for macOS 10.15 Catalina. All future versions will support WebKit for macOS 11 BigSur
```
Comment (mxschmitt) [inline docs/src/release-notes-csharp.md:52]: yey

**Review by mxschmitt — APPROVED** [review]

File: docs/src/release-notes-csharp.md [hunk 2]
```diff
@@ -5,6 +5,64 @@ title: "Release notes"
...
+Playwright will be re-testing the node with the selector `.status` until
+fetched Node has the `"Submitted"` text. It will be re-fetching the node and
+checking it over and over, until the condition is met or until the timeout is
+reached. You can pass this timeout as an option.
+
+Read more in [our documentation](./api/class-playwrightassertions).
+
+### Other Updates
+
+- Web-First assertions
```
Comment (dgozman) [inline docs/src/release-notes-csharp.md:43]: Drop this one.

File: docs/src/release-notes-csharp.md [hunk 3]
```diff
@@ -5,6 +5,64 @@ title: "Release notes"
...
+- New options for methods [`method: Page.screenshot`], [`method: Locator.screenshot`] and [`method: ElementHandle.screenshot`]:
+  * Option `animations: "disabled"` rewinds all CSS animations and transitions to a consistent state
```
Comment (dgozman) [inline docs/src/release-notes-csharp.md:45]: Should be `animations: SomeKind.OfEnum` here.

File: docs/src/release-notes-java.md [hunk 4]
```diff
@@ -5,6 +5,33 @@ title: "Release notes"
 
 <!-- TOC -->
 
+## Version 1.20
+
+### Highlights
+
+- New options for methods [`method: Page.screenshot`], [`method: Locator.screenshot`] and [`method: ElementHandle.screenshot`]:
+  * Option `animations: "disabled"` rewinds all CSS animations and transitions to a consistent state
```
Comment (dgozman) [inline docs/src/release-notes-java.md:13]: Should be `animations: SomeKind.OfEnum` here.

**Review by dgozman — APPROVED** [review]

**Review by dgozman — APPROVED** [review]

**Pull request merged**